api/api.yaml

```diff
@@ -2019,6 +2019,20 @@ paths:
                   type: string
```

Comment:
can you now turn this into an enum field that can have `webhook` or `kinesis` as the values?